<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="say_hello">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Say Hello!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hello_output">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>371</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_input">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>371</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="show_popup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>174</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show popup</string>
    </property>
   </widget>
   <widget class="QLabel" name="actor_msg">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionYeni">
   <property name="text">
    <string>Yeni</string>
   </property>
  </action>
  <action name="actionKaydet">
   <property name="text">
    <string>Kaydet</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>hello_output</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>say_hello</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>hello_output</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
